README.md

```diff
@@ -31,6 +31,45 @@ Please be aware that this older method might leak your credentials when you use
 ## Installation for self-hosting
 If you want run this script yourself, you need a recent Python 3 version and install the packages in `requirements.txt` with `pip install -r requirements.txt`.
 
+## Installation for Docker Compose (self-built image)
+If you want to run this script yourself in a Docker container (using Docker Compose), follow the following steps. This assumes Git and Docker (with Compose plugin) are already installed on your system.
+
+### Step 1  - download copy of this respository
+Go to the directory where you want your local copy, e.g. `cd /home/user/`.
+
+Then, pull this repository.
+`git clone https://github.com/ThijsRay/podimo.git`
+This will download the entire script and all files to a new directory, e.g. /home/user/podimo.
+
+### Step 2  - create a Docker Compose file:
+Go to the directory where you want your docker-compose.yml file to live, e.g. `cd /opt/docker`.
+
+Create a docker-compose.yml file with the following content:
+```
+version: "3"
+services:
+	podimo:
+		build: 
+		context: /home/user/podimo  # Directory from step 1
```

Comment:
Maybe it would be nice to just add the file to add a docker-compose.yml file to the repository, and use relative paths instead. That would save a step in setting it up.